deps/build.jl

```diff
@@ -53,18 +52,18 @@ end
 if HAS_CUDA  # then check cudnn
   let cudnnlib = Libdl.find_library("libcudnn", CUDAPATHS)
     HAS_CUDNN = !isempty(cudnnlib) && Libdl.dlopen_e(cudnnlib) != C_NULL
-    if HAS_CUDNN
-      # TODO: do more version check?
-      cudnnver = dec(ccall((:cudnnGetVersion, cudnnlib), Csize_t, ()))
+    if HAS_CUDNN && !haskey(ENV, "CUDA_HOME")  # inference `CUDA_HOME`
+      ENV["CUDA_HOME"] = dirname(dirname(cudnnlib))
```

Comment:
YFI: I try to guessing your CUDA_HOME via libcudnn's path

Reply:
How about this for guessing the path to CUDA?
`filter(x->contains(x, "cuda"), split(ENV["PATH"], ":"))`
I assume everyone with CUDA installed put it in the PATH variable...

Reply:
🤔 , it does not set `PATH` and `LD_LIBRARY_PATH` on my machine, but works fine.

BTW, where is your libcublas
```
julia> Libdl.dlpath("libcublas")                                                                                                                  
"/opt/cuda/lib64/libcublas.so
```
seesm you have different installation path for CUDA and cuDNN,
maybe I should determin path from cuBLAS?